torchdrivesim/mesh.py

```diff
@@ -799,3 +799,34 @@ def generate_annulus_polygon_mesh(polygon: Tensor, scaling_factor: float, origin
     if category is not None:
         mesh = rendering_mesh(mesh, category=category)
     return mesh
+
+
+def generate_disc_mesh(radius: float = 2, num_triangles: int = 10, device: str = 'cpu') -> Tuple[Tensor, Tensor]:
```

Comment:
Any reason why this isn't returning a `BaseMesh`? I think that would be more explicit.